public/app/features/trails/AutomaticMetricQueries/graph-builders/simple.ts

```diff
@@ -2,10 +2,11 @@ import { PanelBuilders } from '@grafana/scenes';
 
 import { CommonVizParams } from './types';
 
-export function simpleGraphBuilder({ title, unit }: CommonVizParams) {
+export function simpleGraphBuilder({ title, unit, axisLabel }: CommonVizParams) {
   return PanelBuilders.timeseries() //
     .setTitle(title)
     .setUnit(unit)
     .setOption('legend', { showLegend: false })
-    .setCustomFieldConfig('fillOpacity', 9);
+    .setCustomFieldConfig('fillOpacity', 9)
+    .setCustomFieldConfig('axisLabel', axisLabel);
```

Comment:
not sure using an axis label for this is great, esp when the breakdowns don't have it causes the x-axis to be missaligned  beteween main and breakdown

Reply:
Ok. I can bring it back to parentheses for the "main".